getting-started/display.rst

```diff
@@ -0,0 +1,361 @@
+
+.. code-block:: python
+
+   import lsst.afw.display as afwDisplay
+
+The display framework provides a uniform API for multiple display backends, including DS9_ and LSST's Firefly viewer.
+For this tutorial we'll create a display with the ``ds9`` backend:
+
+.. code-block:: python
+
+   display = afwDisplay.getDisplay(backend='ds9')
+
+Display the calexp (calibrated exposure)
+========================================
+
+Then use the display's ``mtv`` method to view the ``calexp`` in DS9:
+
+.. code-block:: python
+
+   display.mtv(calexp)
+
+As soon as you execute the command a single Hyper Suprime-Cam calibrated exposure, the ``{'filter': 'HSC-R', 'visit': 903334, 'ccd': 23}`` dataId, should appear in the DS9 application.
+
+Notice that the DS9 display is filled with colorful regions.
+These are mask regions.
+Each color reflects a different mask bit that correspond to detections and different types of detector artifacts.
+We'll see how to interpret these colors :ref:`later <getting-started-display-mask-colors>`, but first you'll likely want to adjust the image display.
+
+Improving the image display
+===========================
+
+The display framework gives you control over the image display to help bring out image details.
+
+To make masked regions semi-transparent, so that underlying image features are visible, try:
+
+.. code-block:: python
+
+   display.setMaskTransparency(60)
+
+The ``setMaskTransparency`` method's argument can range from ``0`` (fully opaque) to ``100`` (fully transparent).
+
+You can also control the colorbar scaling algorithm with the display's ``scale`` method.
+Try an ``asinh`` stretch with the ``zscale`` algorithm for automatically selecting the white and black points:
+
+.. code-block:: python
+
+   display.scale("asinh", "zscale")
+
+Instead of an automatic algorithm like zscale (or ``minmax``) you can explicitly provide both a minimum (black) and maximum (white) value:
+
+.. code-block:: python
+
+   display.scale("asinh", -1, 30)
+
+.. _getting-started-display-mask-colors:
+
+Interpreting displayed mask colors
+==================================
+
+The display framework renders each plane of the mask in a different color (*plane* being a different bit in the mask).
+To interpret these colors you can get a dictionary of mask planes from the ``calexp`` and query the display for the colors it rendered each mask plane with.
+Run:
+
+.. code-block:: python
+
+   for maskName, maskBit in mask.getMaskPlaneDict().items():
+       print('{}: {}'.format(maskName, display.getMaskPlaneColor(maskName)))
+
+As an example, this result is:
+
+.. code-block:: text
+
+   DETECTED_NEGATIVE: cyan
+   CROSSTALK: None
+   INTRP: green
+   DETECTED: blue
+   UNMASKEDNAN: None
+   NO_DATA: orange
+   BAD: red
+   EDGE: yellow
+   SUSPECT: yellow
+   NOT_DEBLENDED: None
+   CR: magenta
+   SAT: green
+
+Footprints of detected sources are rendered in blue and the saturated cores of bright stars are drawn in green.
```

Comment:
Should "Footprints" link to a definition, e.g.
https://confluence.lsstcorp.org/display/LSWUG/Astro+Glossary#AstroGlossary-Footprint
?